addon/helpers/filestack-image.js

```diff
@@ -14,28 +14,31 @@ export default Helper.extend({
   key: alias('config.filestackKey'),
   compute([token], hash) {
     let options = [];
-    let resizeOptions, urlRoot;
+    let resizeOptions, urlRoot, imageUrl;
     if(!token) {
       return '';
     }
+    if(token.match(/http(s?):\/\//)) {
+      imageUrl = token;
+      urlRoot = `https://process.filestackapi.com/${this.get('key')}`;
+    } else {
+      imageUrl = `https://cdn.filestackcontent.com/${token}`;
+      urlRoot = 'https://process.filestackapi.com';
+    }
     Object.keys(hash).forEach((key) => {
```

Comment:
shouldn't this forEach move to within the if at current line 36?  No need to process if we don't have enough options, right?

Reply:
Yes good point! Updating incoming